utilities/data_simulation/Download_data.py

```diff
@@ -2,7 +2,7 @@
 import os
 import subprocess
 import time
-
+import zenodo_get
```

Comment:
Is this used?

Reply:
It is used here:    

subprocess.check_call(["zenodo_get", 'https://zenodo.org/records/10696605'])

but as supprocess, it is not recognized that it is used. If you remove it, however, this crashes :).

Reply:
I don't understand that, but I guess it's not a big deal.

Reply:
I think this starts a subprocess which then runs the commando in the string (which contains Zenodo_get). But the Zenodo_get function is not recognized by interpreters as it is hidden in a string.
The download happens outside Pycharm, and if I don't put this in a subprocess, it somehow continues the code while downloading, running into errors (using data that still needs downloading).